sqlglot/generator.py

```diff
@@ -589,6 +589,12 @@ def except_sql(self, expression: exp.Except) -> str:
     def except_op(self, expression: exp.Except) -> str:
         return f"EXCEPT{'' if expression.args.get('distinct') else ' ALL'}"
 
+    def exclude_sql(self, expression: exp.Exclude) -> str:
+        return f" EXCLUDE {', '.join(expression.args.get('expressions'))}"
```

Comment:
you don't need a get since we should always have expressions for this and rename